Gemfile

```diff
@@ -4,6 +4,8 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 ruby '3.0.1'
 
 gem 'rspec', '~> 3.0'
```

Comment:
It's not a part of this PR, but `rspec` should be move to group `developement and test`